app/forms/sufia/forms/collection_form.rb

```diff
@@ -1,7 +1,20 @@
 module Sufia::Forms
   class CollectionForm < CurationConcerns::Forms::CollectionEditForm
     def rendered_terms
-      terms - [:visibility]
+      [:title,
+       :creator,
+       :contributor,
+       :description,
+       :tag,
+       :rights,
+       :publisher,
+       :date_created,
+       :subject,
+       :language,
+       :identifier,
+       :based_near,
+       :related_url,
+       :resource_type]
```

Comment:
I'm not sure collections should have resource types, but I don't care strongly about this.